### PR TITLE
ADD notification payload in INFO log traces

### PR DESCRIPTION
**Pull request by fgalan**
Issue #4449 

- [x] Doc (mainly logs.md)
- [x] Test including simplified all the different notification kinds
    - NGSIv2
    - Simplified NGSIv2
    - Custom text-based
    - Empty payload

### CODE REVIEW

File: doc/manuals/admin/logs.md [hunk 1]
```diff
@@ -170,13 +170,13 @@ time=2020-10-26T10:32:41.724Z | lvl=INFO | corr=93bdc5b4-1776-11eb-954d-000c29df
...
 ```
 
 * In the case of MQTT notifications the `msg` field of the above trace is slightly different:
 
 ```
-time=2020-10-26T10:32:22.145Z | lvl=INFO | corr=87f708a8-1776-11eb-b327-000c29df7908; cbnotif=1 | trans=1603707992-318-00000000003 | from=0.0.0.0 | srv=s1| subsrv=/A | comp=Orion | op=logTracing.cpp[63]:logInfoMqttNotification | msg=MQTT Notif delivered (subId: 60ffea6c1bca454f9a64c96c): broker: localhost:1883, topic: sub2
+time=2020-10-26T10:32:22.145Z | lvl=INFO | corr=87f708a8-1776-11eb-b327-000c29df7908; cbnotif=1 | trans=1603707992-318-00000000003 | from=0.0.0.0 | srv=s1| subsrv=/A | comp=Orion | op=logTracing.cpp[63]:logInfoMqttNotification | msg=MQTT Notif delivered (subId: 60ffea6c1bca454f9a64c96c): broker: localhost:1883, topic: sub2, payload (121 bytes): {"subscriptionId":"60ffea6c1bca454f9a64c96c","data":[{"id":"E","type":"T","A":{"type":"Float","value":1,"metadata":{}}}]}
```
Comment (fgalan) [inline doc/manuals/admin/logs.md:179]:
```suggestion
time=2020-10-26T10:32:22.145Z | lvl=INFO | corr=87f708a8-1776-11eb-b327-000c29df7908; cbnotif=1 | trans=1603707992-318-00000000003 | from=0.0.0.0 | srv=s1| subsrv=/A | comp=Orion | op=logTracing.cpp[63]:logInfoMqttNotification | msg=MQTT Notif delivered (subId: 60ffea6c1bca454f9a64c96c): broker: localhost:1883, topic: sub2, payload (123 bytes): {"subscriptionId":"5f914177334436ea590f6edb","data":[{"id":"E","type":"T","A":{"type":"Number","value":42,"metadata":{}}}]}
```

File: doc/manuals/admin/logs.md [hunk 2]
```diff
@@ -170,13 +170,13 @@ time=2020-10-26T10:32:41.724Z | lvl=INFO | corr=93bdc5b4-1776-11eb-954d-000c29df
...
 ```
 
 * In the case of MQTT notifications the `msg` field of the above trace is slightly different:
 
 ```
-time=2020-10-26T10:32:22.145Z | lvl=INFO | corr=87f708a8-1776-11eb-b327-000c29df7908; cbnotif=1 | trans=1603707992-318-00000000003 | from=0.0.0.0 | srv=s1| subsrv=/A | comp=Orion | op=logTracing.cpp[63]:logInfoMqttNotification | msg=MQTT Notif delivered (subId: 60ffea6c1bca454f9a64c96c): broker: localhost:1883, topic: sub2
+time=2020-10-26T10:32:22.145Z | lvl=INFO | corr=87f708a8-1776-11eb-b327-000c29df7908; cbnotif=1 | trans=1603707992-318-00000000003 | from=0.0.0.0 | srv=s1| subsrv=/A | comp=Orion | op=logTracing.cpp[63]:logInfoMqttNotification | msg=MQTT Notif delivered (subId: 60ffea6c1bca454f9a64c96c): broker: localhost:1883, topic: sub2, payload (123 bytes): {"subscriptionId":"5f914177334436ea590f6edb","data":[{"id":"E","type":"T","A":{"type":"Number","value":42,"metadata":{}}}]}
```
Comment (fgalan) [inline doc/manuals/admin/logs.md:179]:
```suggestion
time=2020-10-26T10:32:22.145Z | lvl=INFO | corr=87f708a8-1776-11eb-b327-000c29df7908; cbnotif=1 | trans=1603707992-318-00000000003 | from=0.0.0.0 | srv=s1| subsrv=/A | comp=Orion | op=logTracing.cpp[63]:logInfoMqttNotification | msg=MQTT Notif delivered (subId: 60ffea6c1bca454f9a64c96c): broker: localhost:1883, topic: sub2, payload (123 bytes): {"subscriptionId":"60ffea6c1bca454f9a64c96c","data":[{"id":"E","type":"T","A":{"type":"Number","value":42,"metadata":{}}}]}
```

File: test/functionalTest/cases/0000_deprecated_checkings/log_deprecate_warning.test [hunk 3]
```diff
@@ -31,7 +31,7 @@ brokerStart CB 0 IPV4 -logDeprecate
 
 #
 # 01. Query E1-T1
-# 02. GET /v1/contextEntities/E
+# 02. GET /v1/contextEntities/E/attributes/A
```
Comment (fgalan) [inline test/functionalTest/cases/0000_deprecated_checkings/log_deprecate_warning.test:34]: Just a typo fix (not directly related with this PR).

NTC (informative)

**Review by AlvaroVega — APPROVED** [review]

**Comment by fgalan** [comment]
@fisuda this PR does some modification to English documentation. It would be great if you could sync the Japanese translation in a PR, please. Thanks!

**Comment by fisuda** [comment]
I sent the PR #4461. Thanks.

**Pull request merged**